ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,279 @@
+        mv /etc/elasticsearch/certs/filebeat.pem /etc/filebeat/certs/ 
+        # Start Filebeat
+        startService "filebeat"
+        logger "Done"
+    fi
+}
+
+## Kibana
+installKibana() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Installing Open Distro for Kibana..."
+    $sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q 
+    if [ "$?" != 0 ]; then
+        logger "Error: Kibana installation failed"
+        exit 1;
+    else
+        curl -so /etc/kibana/kibana.yml ${resources_url}/resources/open-distro/kibana/7.x/kibana_all_in_one.yml --max-time 300 
+        echo "telemetry.enabled: false" >> /etc/kibana/kibana.yml
+        cd /usr/share/kibana 
+
+        if [ "${STATUS_PACKAGES}" = "prod" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/${WAZUH_MAJOR}.x/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}.zip 
+        elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-1.zip 
+        fi
+                
+        if [ "$?" != 0 ]
+        then
+            logger "Error: Wazuh Kibana plugin could not be installed."
+            exit 1;
+        fi
+        mkdir -p /etc/kibana/certs 
+        mv /etc/elasticsearch/certs/kibana.key /etc/kibana/certs/ 
+        mv /etc/elasticsearch/certs/kibana.pem /etc/kibana/certs/ 
+        setcap 'cap_net_bind_service=+ep' /usr/share/kibana/node/bin/node 
+
+        # Start Kibana
+        startService "kibana"
+
+        logger "Done"
+    fi
+}
+
+## Health check
+healthCheck() {
+
+    cores=$(nproc)
+    ram_gb=$(free -m | awk '/^Mem:/{print $2}')
+
+    if [[ $cores -lt 2 ]] || [[ $ram_gb -lt 4096 ]]; then
+        logger "The system must have at least 4Gb of RAM and 2 CPUs"
+        exit 1;
+    else
+        logger "Starting the installation..."
+    fi
+}
+
+checkInstallation() {
+
+    logger "Checking the installation..."
+    curl -XGET https://localhost:9200 -uadmin:admin -k --max-time 300 
+    if [ "$?" != 0 ]; then
+        logger "Error: Elasticsearch was not successfully installed."
+        exit 1
+    else
+        logger "Elasticsearch installation succeeded."
+    fi
+    filebeat test output 
+    if [ "$?" != 0 ]; then
+        logger "Error: Filebeat was not successfully installed."
+        exit 1;
+    else
+        logger "Filebeat installation succeeded."
+    fi
+    logger "Initializing Kibana (this may take a while)"
+    until [[ "$(curl -XGET https://localhost/status -I -uadmin:admin -k -s | grep "200 OK")" ]]; do
+        logger -ne $char
+        sleep 10
+    done
+    logger $'\nInstallation finished'
+}
+
+cleanInstall(){
+    rm -rf /etc/yum.repos.d/adoptopenjdk.repo
+    rm -rf /etc/yum.repos.d/opendistroforelasticsearch-artifacts.repo
```

Comment:
This repo is not added anywhere why are you removing it

Reply:
Done in 73ddd7d9a8b4abd89d1689d775f2fda80bdf7de5